.fastlane/beta-post-s3.rb

```diff
@@ -26,10 +26,7 @@ def current_builds
 #
 
 def plist_path
-  @plist_path ||= begin
-    root = File.dirname(File.dirname(__FILE__))
-    File.join(root, 'Kickstarter-iOS', 'Info.plist')
-  end
+  @plist_path ||= File.join('./../', 'Kickstarter-iOS', 'Info.plist')
```

Comment:
these blocks _ought_ to be the same but reverting to the previous implementation because it was giving me trouble